component-lib/typography.html

```diff
@@ -0,0 +1,242 @@
+							<p>Badge</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./button.html">
+							<p>Button</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./cards.html">
+							<p>Card</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./images.html">
+							<p>Image</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./input.html">
+							<p>Input</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./typography.html">
+							<p>Typography</p>
+						</a>
+						<a class="drawer-item link-no-style" href="./modal.html">
+							<p>Modal</p>
+						</a>
+					</div>
+				</div>
+			</div>
+			<div class="main-content">
+				<!-- componen-1 -->
+				<div class="component-container">
+					<div class="component-intro">
+						<div class="title">Typography</div>
+						<div class="sub-title">
+							The Typography component makes it easy to apply a default set of
+							font weights and sizes in your application.
+						</div>
+					</div>
+					<div class="component-example">
+						<div class="title">Heading Typography</div>
+						<div class="sub-title">
+							Use classes from h1 to h6 to acheive different heading style as shown below.
+						</div>
+
+						<div class="component-display-container continer-vertical">
+							<div class="h1">h1. Heading</div>
+							<div class="h2">h2. Heading</div>
+							<div class="h3">h3. Heading</div>
+							<div class="h4">h4. Heading</div>
+							<div class="h5">h5. Heading</div>
+							<div class="h6">h6. Heading</div>
+						</div>
+
+						<div class="code-snippet-display">
+							<iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=solarized+dark&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522h1%2522%253Eh1.%2520Heading%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522h2%2522%253Eh2.%2520Heading%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522h3%2522%253Eh3.%2520Heading%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522h4%2522%253Eh4.%2520Heading%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522h5%2522%253Eh5.%2520Heading%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522h6%2522%253Eh6.%2520Heading%253C%252Fdiv%253E"
+                                style="width: 680px; height: 360px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+						</div>
+					</div>
+					<div class="component-example">
+						<div class="title">Special Typography</div>
+						<div class="sub-title">
+							Use respective classes to acheive special Typography such highlighted text, text with line through, label text, etc.
+						</div>
+
+						<div class="component-display-container continer-vertical">
```

Comment:
please check `continer-vertical` spelling it will couse error in future

Reply:
I know it. But if I change it now I have to change at lots of places. So as soon I complete my component library I will fix that spelling.